jenkins.go

```diff
@@ -503,7 +512,7 @@ func CreateJenkins(base string, auth ...interface{}) *Jenkins {
 		base = base[:len(base)-1]
 	}
 	j.Server = base
-	j.Requester = &Requester{Base: base, SslVerify: false}
+	j.Requester = &Requester{Base: base, SslVerify: true}
```

Comment:
Because that is highly unusual. Most programs that use self-signed certificate, don't want to affect every other program on the system to trust that certificate.